sklearn/linear_model/_base.py

```diff
@@ -419,6 +412,27 @@ class LinearRegression(MultiOutputMixin, RegressorMixin, LinearModel):
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
+    normalize : bool, default=False
+        This parameter is ignored when ``fit_intercept`` is set to False.
```

Comment:
We usually put these considerations at the end

Reply:
@NicolasHug after synchronizing with upstream considerations about `normalize` has been moved. It isn't clear to me if there is still an issue here... do you mind having a look again? Thanks!